code/style.css

```diff
@@ -1,3 +1,178 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #f8e2cf;
+  display: flex;
+  flex-direction: column;
+  color: black;
+  font-family: "Montserrat", sans-serif;
+}
+
+#user-info {
+  display: flex;
+  justify-content: center;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 30%;
+  height: 30%;
+  align-self: center;
+  box-shadow: 8px 8px 8px rgb(114, 114, 114);
+}
+
+#profile-name {
+  flex-direction: column;
+  margin-left: 20px;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+}
+
+.github-icon {
+  width: 20%;
+  height: auto;
+}
+
+h1 {
+  align-self: center;
+  font-size: 25px;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h3 {
+  font-size: 12px;
+}
+
+#projects {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
This is a solution that would have solved all my grid issues for the project. Will save it for my next ones!